libs/components/ag-grid/src/lib/modules/ag-grid/types/sky-grid-options.ts

```diff
@@ -1,10 +1,10 @@
 import { GridOptions } from 'ag-grid-community';
 
-export interface SkyGetGridOptionsArgs {
+export interface SkyGetGridOptionsArgs<T = any> {
```

Comment:
Can this be `unknown` instead of `any`? Or would that break backwards compatibility?